pyprof/prof/embedding.py

```diff
@@ -28,56 +26,46 @@ def __init__(self, d):
         op = marker['op']
         args = marker['args']
 
-        self.marker = marker
-        self.mod_ = mod
-        self.op_ = op
-        self.args = args
+        self._mod = mod
+        self._op = op
 
         assert (mod == "torch.nn.functional")
         assert (op == "embedding")
 
-        self.ishape = args[0]['shape']
-        self.itype = args[0]['dtype']
+        inp = args[0]
```

Comment:
`inp` and `emb` are not self explaining names. Can you expand the names to improve readability?